### PR TITLE
chore: update outdated dev dependencies

### PR DESCRIPTION
**Pull request by SimenB**
Closes #562
Closes #529
Closes #567
Closes #526
Closes #547
Closes #554
Closes #528

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "prepare": "yarn build && yarn postbuild",
     "lint": "eslint . --ignore-pattern '!.eslintrc.js' --ext js,ts",
-    "prettylint": "prettylint docs/**/*.md README.md package.json",
+    "prettylint": "prettier --check docs/**/*.md README.md package.json",
```
Comment (G-Rath) [inline package.json:31]: nit: should we maybe change the name?

File: package.json [hunk 2]
```diff
@@ -47,47 +47,42 @@
...
+    "husky": "^4.2.5",
+    "jest": "^26.0.1",
+    "jest-runner-eslint": "^0.8.0",
+    "lint-staged": "^10.2.2",
+    "prettier": "^2.0.5",
     "resolve-from": "^5.0.0",
     "rimraf": "^3.0.0",
-    "semantic-release": "^15.13.28",
+    "semantic-release": "^17.0.7",
     "typescript": "^3.5.3"
```
Comment (G-Rath) [inline package.json:74]: nit: Would be cool to bump typescript - while symbolic, I feel it's nice to have it at a higher number.

File: package.json (hunk 1 above)
Comment (SimenB) [reply package.json]: sure

File: package.json (hunk 2 above)
Comment (SimenB) [reply package.json:75]: it's bumped to latest in the lockfile

**Comment by SimenB** [comment]
This is going into `next`, so I think it'll conflict regardless

**Comment by G-Rath** [comment]
> This is going into next

Woops completely missed that - makes a lot more sense now :joy:

**Pull request merged**